packages/dht/src/connection/websocket/WebsocketServerConnection.ts

```diff
@@ -26,14 +26,19 @@ export class WebsocketServerConnection extends EventEmitter<ConnectionEvents> im
 
         this.resourceURL = resourceURL
         this.connectionId = createRandomConnectionId()
-        this.remoteIpAddress = remoteAddress
+        this.remoteAddress = remoteAddress
 
         socket.on('message', this.onMessage)
         socket.on('close', this.onClose)
         socket.on('error', this.onError)
 
         this.socket = socket
     }
+    
+    // use a getter to make it possible to mock the value in tests
```

Comment:
The value is not mocked in any tests so could just revert the change

Reply:
Yes it is mocked, for example here: https://github.com/streamr-dev/network/blob/7284f0195dd7380fe8095c0b8026d0e3a03f2ae7/packages/dht/test/end-to-end/GeoIpLayer0.test.ts#L34